src/thunderbots/software/ai/world/world.h

```diff
@@ -107,9 +116,17 @@ class World final
      */
     Team& mutableEnemyTeam();
 
+    /**
+     * Returns a const reference to the Game State
+     *
+     * @return a const reference to the Game State
+     */
+     const GameState& gameState() const;
+
```

Comment:
We will probably want a `mutableGameState` function as well, similar to the other members of the `World` class so that the `ai` class can call the `gameState's` `update` function